.circleci/config.yml

```diff
@@ -98,8 +98,9 @@ jobs:
           export ENV_NAMESPACE=paymenthub
           kubectl delete secret bulk-processor-secret -n $ENV_NAMESPACE || echo "delete the secret if exist"
           kubectl create secret generic bulk-processor-secret \
-          --from-literal=aws-access-key="$S3_ACCESS_KEY_ID" \
-          --from-literal=aws-secret-key="$S3_SECRET_ACCESS_KEY" -n $ENV_NAMESPACE
+          --from-literal=aws-access-key="$MINIO_ROOT_USER" \
```

Comment:
these steps can be removed. as we are setting the default minio credentials within the base chart itself.

Reply:
done